build/ToolsetPackages/RoslynToolset.csproj

```diff
@@ -9,7 +9,6 @@
     <PackageReference Include="Microsoft.Build.Logging.StructuredLogger" Version="$(MicrosoftBuildLoggingStructuredLoggerVersion)" ExcludeAssets="all" />
     <PackageReference Include="Microsoft.NETCore.Platforms" Version="$(MicrosoftNETCorePlatformsVersion)" ExcludeAssets="all" />
     <PackageReference Include="Microsoft.DotNet.Build.Tasks.Feed" Version="$(MicrosoftDotNetBuildTasksFeedVersion)" ExcludeAssets="all" />
-    <PackageReference Include="Microsoft.Net.RoslynDiagnostics" Version="$(MicrosoftNetRoslynDiagnosticsVersion)" ExcludeAssets="all" />
```

Comment:
Can you file an issue on roslyn-analyzers to delete `microsoft.net.roslyndiagnostics` package? Not sure if any of our other repos are using it.

Reply:
➡️ dotnet/roslyn-analyzers#1750